spicy/toolchain/src/compiler/printer.cc

```diff
@@ -28,6 +30,18 @@ struct VisitorPrinter : visitor::PreOrder {
         result = true;
     }
 
+    void operator()(hilti::type::StrongReference* n) final {
+        if ( auto m = n->parent<hilti::declaration::Module>(); m && m->uid().process_extension != ".spicy" )
+            return;
+
+        if ( n->isWildcard() )
+            out << "T&";
+        else
+            out << n->dereferencedType() << "&";
```

Comment:
While looking at zeek/zeek#3685 I had seen error messages involving `strong_ref<strong_ref<T>>` and am wondering whether we could now produce nonsensical `T&&`, but it seems the other cleanups in the PR make that much less likely.